content/scout/quickstart.md

```diff
@@ -108,126 +83,187 @@ the underlying vulnerable express version to 4.17.3 or later.
 
-4. Push it to Docker Hub using the new version tag:
+  No vulnerable packages detected
 
-   ```console
-   $ docker push <ORG_NAME>/scout-demo:v3
-   ```
+```
 
-5. Select the new image tag in Docker Desktop or the Docker Scout Dashboard and you
-   can see that the base image has been updated, removing many vulnerabilities.
+## Step 5: Evaluate policy compliance
 
-   You can see the same using the Docker CLI command:
+While inspecting vulnerabilities based on specific packages can be useful,
+it isn't the most effective way to improve your supply chain conduct.
 
-   ```console
-   $ docker scout cves <ORG_NAME>/scout-demo:v3
-   ```
+Docker Scout also supports policy evaluation,
+a higher-level concept for detecting and fixing issues in your images.
+Policies are a set of customizable rules that lets organizations track whether
+images are compliant with their supply chain requirements.
 
-## Step 6: Collaborate on vulnerabilities
+Because policy rules are specific to each organization,
+you must specify which organization's policy you're evaluating against.
+Use the `docker scout config` command to configure your Docker organization.
 
-You can see and share the same vulnerability information about an image and
-the other images in your organization in the [Docker Scout Dashboard](./dashboard.md).
+```console
+$ docker scout config organization <ORG_NAME>
+    ✓ Successfully set organization to <ORG_NAME>
+```
 
-All organization members can see an overview of all their images from integrated container registries,
-and get remediation advice at their fingertips. This helps team members in
-security, compliance, and operations to know what vulnerabilities and issues to focus on.
+Now you can run the `quickview` command to get an overview
+of the compliance status for the image you just built,
+using the default, out-of-the-box policy configuration.
 
-1. Select the **Images** tab on the [Docker Scout Dashboard](https://scout.docker.com).
-2. Select any of the tags under
-   the **Most Recent Image** column, and you can see the same vulnerability
-   information as you saw in Docker Desktop and the Docker CLI and share this link with anyone else in your organization.
+```console
+$ docker scout quickview
+
+...
+Policy status  FAILED  (2/7 policies met, 2 missing data)
+
+  Status │                  Policy                   │           Results
+─────────┼───────────────────────────────────────────┼──────────────────────────────
+  ✓      │ Copyleft licenses                         │    0 packages
+  !      │ Critical vulnerabilities                  │    2C     0H     0M     0L
+  !      │ Default non-root user                     │
+  !      │ Fixable critical and high vulnerabilities │    2C    16H     0M     0L
+  ✓      │ High-profile vulnerabilities              │    0C     0H     0M     0L
+  ?      │ Outdated base images                      │    No data
+  ?      │ Supply chain attestations                 │    No data
```

Comment:
After reading section 5, as a new user I was thinking:

- Where are these policies coming from? 
- How can I create custom ones?

It'd be great to add a small paragraph to cover those, e.g:


"Docker Scout provides a set of out of the box policies (link to https://docs.docker.com/scout/policy/#out-of-the-box-policies) enabled by default in your organisation. You can customise them or disable them to fit your needs in scout.docker.com".

Reply:
I have more or less that before the quickview output. Let me rephrase it slightly.